### DISCUSSION TITLE
autogen not picking up a tool call?

### DISCUSSION THREAD

**Discussion by invokeinnovation** [microsoft/autogen#2317] [Q&A]
User (to Assistant):
 What is (44232 + 13312 / (232 - 32)) * 5?

--------------------------------------------------------------------------------
No default IOStream has been set, defaulting to IOConsole.

>>>>>>>> USING AUTO REPLY...
No default IOStream has been set, defaulting to IOConsole.
No default IOStream has been set, defaulting to IOConsole.
Assistant (to User):

{"name": "calculator", "arguments": {"a": 44232, "b": 13312, "operator": " +"}}

--------------------------------------------------------------------------------

**Comment by ekzhu** [comment]
> {"name": "calculator", "arguments": {"a": 44232, "b": 13312, "operator": " +"}}

This is the tool call request. What's your LLM config and your tool executor agent setup?
